plugin/teksi_wastewater/teksi_wastewater_plugin.py

```diff
@@ -82,6 +83,17 @@ class TeksiWastewaterPlugin:
     profile = None
 
     def __init__(self, iface):
+        if os.environ.get("QGIS_DEBUGPY_HAS_LOADED") is None:
```

Comment:
I'm not sure including debugpy in production environment is a good practice. 
Is there a way to include it only in a dev mode only?

Reply:
There is an existing plugin setting that we can use:
```python
self.settings.value("/TWW/DeveloperMode", False, type=bool)
```

![image](https://github.com/user-attachments/assets/e25253cf-eea2-44da-91a6-f0e37aa6a4e4)

Reply:
@cymed Können wir die Anpassungen noch weiter verfolgen?

Reply:
I will extend the if statement to include the Developer mode setting.